flags.go

```diff
@@ -0,0 +1,113 @@
+package main
```

Comment:
I wouldn't mind this being its own package

`pkg/flagutil` maybe?

Reply:
Sure.